lib/rollbar/encoding/encoder.rb

```diff
@@ -0,0 +1,54 @@
+module Rollbar
+  module Encoding
+    class Encoder
+      ALL_ENCODINGS = [::Encoding::UTF_8, ::Encoding::ISO_8859_1, ::Encoding::ASCII_8BIT, ::Encoding::US_ASCII]
+      ASCII_ENCODINGS = [::Encoding::US_ASCII, ::Encoding::ASCII_8BIT, ::Encoding::ISO_8859_1]
+      ENCODING_OPTIONS = { :invalid => :replace, :undef => :replace, :replace => '' }
+
+      attr_accessor :object
+
+      def initialize(object)
+        @object = object
+      end
+
+      def encode
+        value = object.to_s
+
+        encoded_value = force_encoding(value).encode(*encoding_args(value))
+
+        object.is_a?(Symbol) ? encoded_value.to_sym : encoded_value
```

Comment:
This may be a dumb question, but what is this line for? Why do we need to turn the string back into a symbol?